style.css

```diff
@@ -483,3 +494,466 @@ footer .bottom_line {
   margin-top: 1rem;
   margin-bottom: 1rem;
 }
+
+@media all and (min-width: 768px) {
+  header {
+    height: 4.5rem;
+  }
+
+  .headline {
+    flex-direction: row-reverse;
+    justify-content: flex-start;
+    gap: 5.5rem;
+    padding-top: 12rem;
+    padding-bottom: 16rem;
+    background-image: url(images_4/Illustration-header.png);
+    background-position: top right;
+    background-repeat: no-repeat;
+  }
+
+  .toolbar {
+    display: none;
+  }
+
+  .open_menu {
+    width: 85%;
+    height: auto;
+    display: flex;
+    justify-content: space-between;
+    place-items: center;
+  }
+
+  ul {
+    flex-direction: column;
+    gap: 1rem;
+    margin-left: 2.25rem;
+    padding-top: 5.5rem;
+  }
+
+  footer ul {
+    margin: 0;
+    padding: 1.5rem;
+    flex-direction: row;
+  }
+
+  .open_menu ul {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .open_menu a {
+    color: #fff;
+    text-decoration: none;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 15px;
+    line-height: 133%;
+  }
+
+  .intro {
+    padding: 0;
+    margin-right: 50%;
+  }
+
+  h1 {
+    height: auto;
+    font-family: 'Crete Round', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 72px;
+    line-height: 113%;
+    letter-spacing: 0.37px;
+    color: #f55800;
+  }
+
+  h2 {
+    margin: 1rem 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 138%;
+    color: #fff;
+  }
+
+  p {
+    height: auto;
+    padding: 1.5rem 0 1rem 0;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 160%;
+    color: #f4f5f7;
+  }
+
+  .connect_with {
+    width: fit-content;
+    background-image: none;
+  }
+
+  .works {
```

Comment:
- [ ] Kindly make sure your website is totally responsive on the desktop version, as you can see below the works section is crossing the container at some point, you can use percentages to make sure itis scaling correctly:
![Screenshot from 2023-01-20 06-15-38](https://user-images.githubusercontent.com/97834160/213617260-b7397f43-551b-4bd4-a438-0f40f2ed95ba.png)